pmm-tests/check_upgrade.sh

```diff
@@ -51,9 +51,14 @@ if [ "$DISTRIBUTION" == "ami" ]; then
 		rpm -qa | grep pmm-dump-$PMM_VERSION
 		sudo supervisorctl status | grep victoriametrics | grep RUNNING
 		sudo supervisorctl status | grep vmalert | grep RUNNING
-		grafana-cli plugins ls | grep "vertamedia-clickhouse-datasource @ 2.4.4"
-		grafana-cli plugins ls | grep alexanderzobnin-zabbix-app
 		sudo victoriametrics --version | grep pmm-6401-v1.77.1
+	  if [ $(echo "$PMM_VERSION" | awk -F. '{print $2}') -lt 39 ]; then
+  		grafana-cli plugins ls | grep "vertamedia-clickhouse-datasource @ 2.4.4"
+	  	grafana-cli plugins ls | grep alexanderzobnin-zabbix-app
+    else
```

Comment:
sorry about the nitpick :), seems like the else block requires indenting

Reply:
Yeah! Good catch. There are mixed indents in the file. I've converted all indents to whitespaces.